internal/platform/auth/midware.go

```diff
@@ -32,6 +39,14 @@ func Midware(publicKeyBase64Str string) (app.Middleware, error) {
 			// Extract the token from the Authorization header provided on the request.
 			tokenString := c.Request.Header.Get("Authorization")
 
+			if tokenString == "" && config.AllowQueryString {
+
+				// In the event that the request does not have a header key for the
+				// Authorization header, then we need to try and access it from the URL
+				// query parameters if this was enabled from our config.
+				tokenString = c.Request.URL.Query().Get("access_token")
+			}
+
 			if tokenString == "" {
```

Comment:
That's actually the defined behavior. In the event that the `config.AllowQueryString` is false, then we don't want to access the access_token from the query string at all in the event that the header is not specified. It is preferred (and I can add additional documentation) that the token comes in on the header rather than the query string. So we should only pull the `access_token` from the query string if the feature is enabled and the `tokenString == ""`.